installation/kubernetes/helm/openclarity/values.yaml

```
@@ -916,9 +916,11 @@ crDiscoveryServer:
     digest: ""
     # -- Container Runtime Discovery Server image pull policy
     pullPolicy: IfNotPresent
-  env:
-    name: CONTAINERD_SOCK_ADDRESS
-    value: /run/k3s/containerd/containerd.sock
+
+  # -- Environment variables to set in the Container Runtime Discovery Server container
+  env: []
+  # - name: CONTAINERD_SOCK_ADDRESS
+  #   value: /run/k3s/containerd/containerd.sock
```

lets just ref proper default value here
```suggestion
  #   value: /var/run/containerd/containerd.sock
```